### PR TITLE
Form setup

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
Form setup ready.  Doing pair programming, we add the form section and the footer.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -347,3 +349,108 @@ li {
...
+.contact-ill {
+  position: relative;
+  top: -50%;
+  right: -42vw;
+}
+
+.last-link {
+  display: flex;
+
+  /* justify-content: flex-start; */
```
Comment (huseyinbiyik) [inline styles.css]: Kindly don't leave any commented-out code, so your codebase will be neat and look more professional.👨‍💼

File: styles.css [hunk 2]
```diff
@@ -347,3 +349,108 @@ li {
...
+#submit {
+  padding: 12px;
+  gap: 10px;
+  width: 129px;
+  max-height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: #fff;
+  margin-bottom: 50px;
+}
```
Comment (huseyinbiyik) [inline styles.css]: Very good implementation of the given design but let's add some interaction and add hover and press state like on the design template. 
Screenshot 2023-01-17 at 11 52 17 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Alejandro-Bernal-M  and @matee-safi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] Some of your image files are on the root folder. It is better to put under a folder like your other images. So your repo will look more organized. 👌

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alejandro-Bernal-M** [comment]
Hi @huseyinbiyik , thank you for the corrections. I changed the styles on the buttons, adding the :hover and :active pseudo-classes. And I put all the images into one folder. I'm asking for the other review.

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Alejandro-Bernal-M 

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**